packages/usa-tooltip/src/index.js

```diff
@@ -388,6 +388,16 @@ const tooltip = behavior(
     init(root) {
       selectOrMatches(TOOLTIP, root).forEach((tooltipTrigger) => {
         setUpAttributes(tooltipTrigger);
+
+        const { body, wrapper } = getTooltipElements(tooltipTrigger);
+        wrapper.addEventListener("mouseleave", () => {
+          hideToolTip(body);
+        });
```

Comment:
Follows the same pattern in teardown line `400`.

```suggestion
        wrapper.addEventListener("mouseleave", () => hideToolTip(body));
```

Reply:
Updated in [d4ca8c3](https://github.com/uswds/uswds/pull/5919/commits/d4ca8c3f3f85bb9667a667ff84290aa2d7021346)